pom.xml

```diff
@@ -350,6 +350,18 @@
       </dependency>
 
       <dependency>
+        <groupId>net.jpountz.lz4</groupId>
```

Comment:
I'd make these guys optional, to avoid having to have these deps when you don't use them